src/lib/certificate-manager.ts

```diff
@@ -235,8 +236,10 @@ export async function generateSiteCertificate(
 	domain: string
 ): Promise< { cert: string; key: string } > {
 	try {
-		const siteCertPath = path.join( CERT_DIRECTORY, 'domains', `${ domain }.crt` );
-		const siteKeyPath = path.join( CERT_DIRECTORY, 'domains', `${ domain }.key` );
+		// Convert domain to punycode for certificate generation
```

Comment:
Let's either jsut remove this comment or create a function with wrapping `domainToASCII` + adding JSDoc with explanation why do we need it.
Why:
1. now we have the comment only in 1 place, so if we keep it then it's necessary to copy-paster everywhere
2. The current comment is slightly unclear - how punycode influence in a good way on certificate generation.

Reply:
Do we even need this comment? It seems clear what the line is doing.

Reply:
Agree, let's just remove it 👍